src/plugins/settings/settings.icon.tsx

```diff
@@ -2,7 +2,7 @@
 import React from 'jsx-dom';
 
 export const SettingsIcon = (): SVGElement => (
-  <svg xmlns="http://www.w3.org/2000/svg" fill="#1f1f1f" stroke="#1f1f1f" stroke-width=".5" viewBox="0 0 24 24">
+  <svg id={'settings-icon'} xmlns="http://www.w3.org/2000/svg" fill="#1f1f1f" stroke="#1f1f1f" stroke-width=".5" viewBox="0 0 24 24">
```

Comment:
UIPlayground is not limited to one per page, so we should use a class instead of a unique identifier in case we want to hide the icon.